src/crimson/os/seastore/seastore_types.h

```diff
@@ -36,29 +36,139 @@ struct seastore_meta_t {
 };
 
 // Identifies segment location on disk, see SegmentManager,
```

Comment:
This comment referred to segment_id_t.  Please add a comment like this for every type you introduce below or elsewhere in this PR.

Reply:
will do